README.md

```diff
@@ -9,17 +8,54 @@ Monitore your websites availability, http status code (current and history), cer
 * [Prometheus blackbox exporter](https://github.com/prometheus/blackbox_exporter)
 * [Grafana](https://github.com/grafana/grafana)
 
-## Dependencies
+## Deploying
 
-* [docker](https://docs.docker.com/install/)
-* [docker-composer](https://docs.docker.com/compose/install/)
+- Kubernetes
```

Comment:
How about using three-level titles instead of lists? The format maybe look better.

Reply:
ok, I'll optimize the layout.